.github/workflows/release.yml

```
@@ -0,0 +1,377 @@
+    runs-on: ubuntu-latest
+    needs: [create-release]
+    if: needs.create-release.result == 'success'
+    
+    steps:
+    - name: チェックアウト
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        
+    - name: 次期開発バージョン準備
+      run: |
```

actionlint (1.7.10)

property "pre-release-validation" is not defined in object type {create-release: {outputs: {}; result: string}}

(expression)